src/app/public/modules/repeater/repeater-item.component.scss

```diff
@@ -11,6 +11,7 @@
 }
 
 .sky-repeater-item-right {
+  max-width: 100%;
```

Comment:
This works for a child element that handles its own overflow (for example, the code block includes its own scrollbars), but it would be great if this branch added scrollbars to any content that overflowed the width of the repeater.

Reply:
Yup :D working on that change right now! Ty